### ISSUE TITLE
SSH key commit signing via 1password not working

### ISSUE THREAD

**Issue by jgoz** [ForkIssues/Tracker#1688]
I configured git to use ssh key-based commit signing via [1password](https://blog.1password.com/git-commit-signing/) and I'm getting an error inside Fork on commit:

```
error: Error: AppError { error: Os { code: 2, kind: NotFound, message: "No such file or directory" }, location: Location { file: "ssh/op-ssh-sign/src/ops.rs", line: 41, col: 52 } }

fatal: failed to write commit object
```

This is with fork 2.21 and it happens whether I use the built-in git (2.36.1) or my system git (2.37.1).

**Comment by DanPristupov** [comment]
1. Have you read that? https://developer.1password.com/docs/ssh/agent/compatibility#fork
2. Try to start Fork from the terminal using 'open -a Fork .' command.

**Comment by jgoz** [comment]
Thanks for the reply.

> 1\. Have you read that? https://developer.1password.com/docs/ssh/agent/compatibility#fork

Yes, it looks like Fork is supposed to be compatible as long as I'm using the correct git binary, which I am (2.37.1).

> 2\. Try to start Fork from the terminal using 'open -a Fork .' command.

I tried this and get the same error message as before.

Here is my gitconfig:

```
[user]
	signingkey = ssh-ed25519 <snip>
[commit]
	gpgsign = true
[gpg]
	format = ssh
[gpg "ssh"]
	program = "/Applications/1Password.app/Contents/MacOS/op-ssh-sign"
```

**Comment by jgoz** [comment]
I figured out what the problem was. I was previously using GPG Suite on macos to sign my commits but then switched to 1Password and updated my git config accordingly. But Fork was still trying to use my old signing key via `-c gpg.signingkey` and was also setting `-c gpg.program "/Applications/Fork.app/Contents/Resources/gpg.sh"`.

The root cause is that the old PGP key was being stored in each repository inside `.git/fork-settings`. I simply deleted all of those `fork-settings` files and everything started working.
